backend/webapp/urls.py

```diff
@@ -19,6 +19,7 @@
 router.register(r'priorities', GroupPriorityViewset)
 router.register(r'descriptors', DescriptorViewset)
 router.register(r'baselayergroups', BaseLayerGroupViewset)
+router.register(r'serializedhashstates', SerializedHashstateViewset)
```

Comment:
I'm in favour of renaming that; it's quite "literal".  Even "savedstate" is probably more explanatory.